src/libraries/System.Net.NetworkInformation/src/System/Net/NetworkInformation/AndroidIPGlobalProperties.cs

```diff
@@ -0,0 +1,32 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
I believe the naming convention is `IPGlobalProperties.Android.cs`

Reply:
The other platform-specific files in this project are `LinuxIPGlobalProperties.cs`, `UnixIPGlobalProperties.cs`, and `BsdIPGlobalProperties.cs`, so I replicated it with the `Android` prefix.